ovn_cluster.sh

```diff
@@ -588,6 +657,7 @@ function start() {
         ${RUNC_CMD} exec ${name} ${OVNCTL_PATH} start_controller ${SSL_ARGS}
     done
 
+    # FIXME for ovn-ic (it is not executed for ovn-heater)
```

Comment:
This is actually quite bad.  We can't start a default ovn-fake-multinode cluster.  If I do:

```
CENTRAL_COUNT=2 ./ovn_cluster.sh start
```

This will incorrectly call configure-ovn with all central IPs as arguments:
```
configure-ovn yes ssl:170.168.0.2:6642 ssl:170.168.0.3:6642 no system
```

While the "configure-ovn" function only expects 4 arguments.

Reply:
@dceara ack, I will fix it